backend/src/main/scala/cromwell/backend/standard/StandardCachingActorHelper.scala

```diff
@@ -0,0 +1,89 @@
+package cromwell.backend.standard
+
+import java.nio.file.Path
+
+import akka.actor.{Actor, ActorRef}
+import cromwell.backend._
+import cromwell.backend.callcaching.JobCachingActorHelper
+import cromwell.backend.io.{JobPaths, WorkflowPaths}
+import cromwell.backend.validation.{RuntimeAttributesValidation, ValidatedRuntimeAttributes}
+import cromwell.core.logging.JobLogging
+import wdl4s.TaskCall
+
+import scala.util.Try
+
+/**
+  * Extends the JobCachingActorHelper with standard implementations.
+  *
+  * Like the JobCachingActorHelper, this trait should be extended by a backend, and then __that__ trait should be mixed
+  * into a async job execution actor, and cache hit copying actor.
+  */
+trait StandardCachingActorHelper extends JobCachingActorHelper {
+  this: Actor with JobLogging =>
+
+  def backendInitializationDataOption: Option[BackendInitializationData]
+
+  /** Typed backend initialization. */
+  def backendInitializationDataAs[A <: BackendInitializationData]: A =
+    BackendInitializationData.as[A](backendInitializationDataOption)
+
+  /**
+    * Returns the service registry actor. Both the StandardAsyncExecutorActor and StandardCacheHitCopyingActor traits
```

Comment:
It's a pointer to keep my head straight when I come back to this code in a few months.